### ISSUE TITLE
Add a way to view a plugin's page from the manager.

### ISSUE THREAD

**Issue by rhystedstone** [Macro-Deck-org/Macro-Deck#280]
**Is your feature request related to a problem? Please describe.**
The plugin manager currently has no link to a plugin's GitHub page, meaning additional information, bugtrackers, etc... are not easily accessible.
![image](https://user-images.githubusercontent.com/12123847/184516024-afc0f1bf-ca66-4871-8235-b27bea56fff7.png)

Also, the extension repository hides installed plugins, preventing you from simply looking it up.

**Describe the solution you'd like**
Simply put, some way to open the plugin's page.
Either by a dedicated button, or by clicking on the extension's information box.

Thank you :)
